modules/social_features/social_activity/social_activity.tokens.inc

```diff
@@ -45,8 +50,8 @@ function social_activity_tokens($type, $tokens, array $data, array $options, Bub
             $replacements[$original] = '';
             $summary = '';
 
-            $target_type = $message->field_message_related_object->target_type;
-            $target_id = $message->field_message_related_object->target_id;
+            $target_type = $message->get('field_message_related_object')->getValue()[0]['target_type'];
```

Comment:
//cc @ronaldtebrake lets check if we need to update PHP Stan to ensure we don't get any issues when using 
`            $target_type = $message->field_message_related_object->target_type;`